data/json/items/comestibles/carnivore.json

```
@@ -109,6 +109,32 @@
     "delete": { "flags": [ "SMOKABLE" ] },
     "vitamins": [ [ "vitA", 0 ], [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ], [ "vitB", 0 ] ]
   },
+  {
+    "id": "mutant_meat",
+    "type": "COMESTIBLE",
+    "copy-from": "meat",
+    "name": "chunk of mutant meat",
+    "name_plural": "chunks of mutant meat",
+    "description": "Meat from a heavily mutated animal.  It has an unsettling loose and spongy texture, but smells… mostly normal.  There are strange tangles and formations in it that don't appear natural at all: bits of bone and hair crusted up inside the muscle, as if trying to form another organism.  Still, seems safe to eat, if you cook it and remove the worst parts.",
```

```suggestion
    "description": "Meat from a heavily mutated animal.  It has an unsettling loose and spongy texture, but smells… mostly normal.  There are strange tangles and formations in it that don't appear natural at all: bits of bone and hair crusted up inside the muscle, as if trying to form another organism.  You can probably eat it, if you cook it and remove the worst parts.",
```
    
Could you reword it to not say `safe to eat` since it's not? Same for meat scrap.

---

Does replacing `safe to eat` with `digestible at least` in both cases sound good? I don't want to scare players off too badly, since it _is_ completely safe in moderation.

---

Yep